integreat_cms/cms/templates/events/event_form.html

```diff
@@ -67,7 +73,8 @@ <h1 class="heading overflow-hidden text-ellipsis">
                     <div class="w-full p-4 flex flex-col flex-auto">
                         <div class="flex justify-between">
                             <label for="{{ event_translation_form.title.id_for_label }}"
-                                   data-slugify-url="{% url 'slugify_ajax' region_slug=request.region.slug language_slug=language.slug model_type='event' %}{% if event_form.instance.id %}?model_id={{ event_form.instance.id }}{% endif %}">
+                                   data-slugify-url="{% url 'slugify_ajax' region_slug=request.region.slug language_slug=language.slug model_type='event' %}"
```

Comment:
'if' was necessary for the case where a new event is created.
there is a failure without if